definitions/consented/json/CaseEvent/CaseEvent.json

```diff
@@ -1016,7 +1016,7 @@
     "Name": "Upload Approved Order",
     "Description": "Upload Approved Order",
     "DisplayOrder": 62,
-    "PreConditionState(s)": "referredToJudge",
+    "PreConditionState(s)": "referredToJudge;readyForHearing",
```

Comment:
AC2 awaiting Judicial response state is not used here

Reply:
Also AC3 is not covered here

Reply:
In state Id do not match with name
  "ID": "referredToJudge",
    "Name": "Awaiting Judicial Response",

Reply:
AC3 is already existing functionality